### ISSUE TITLE
Inconsistent tabbing behavior on checkout page

### ISSUE THREAD

**Issue by tflight** [hps/heartland-woocommerce-plugin#186]
When entering the checkout page, when you **first** enter your credit card number and press `tab`, the cursor focus goes seemingly nowhere. For users using assistive technology no focus is indicated. Pressing `tab` a second time will put the focus/cursor into the Expiration Date field. The same thing happens when you **first** try to tab from the Expiration Date field to the Security Code field.

(Note, once you tab through these fields once, if you go back to say the credit card number field, tabbing works as you would expect.

**Comment by securesubmit-buildmaster** [comment]
Hi @tflight. Thank you for reaching out. We're currently investigating this and will review in due course.

**Comment by securesubmit-buildmaster** [comment]
@tflight , we've been unable to duplicate the behavior you described. Tabbing and field focus seem to work as desired/expected regardless of how we go about things. Can you confirm you're using the latest version of the plugin, v2.0.2? If so, if you'd like to point us to your site so that we can have a look we'd be happy to.

**Comment by tflight** [comment]
Yes, latest version of the plugin. Seems to happen with Webkit based browsers (Safari, etc). Can you try with Safari? I can see the behavior in Safari 15.6.1 but not in Firefox nor Chrome.
